src/vcpkg/postbuildlint.cpp

```diff
@@ -16,9 +16,11 @@ using vcpkg::Build::PreBuildInfo;
 
 namespace vcpkg::PostBuildLint
 {
-    static auto not_extension_pred(const Filesystem& fs, const std::string& ext)
+    static auto not_extension_pred(StringView ext)
     {
-        return [&fs, ext](const path& target) { return fs.is_directory(target) || target.extension() != ext; };
+        return [ext](const path& target) {
+            return !Strings::case_insensitive_ascii_equals(vcpkg::u8string(target.extension()), ext);
```

Comment:
1. Should this be system specific?
2. Should this be deduplicated with the extension comparison elsewhere (playfab)?